.github/workflows/ci.yml

```diff
@@ -21,12 +21,29 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: |
+          node_modules
+          .cache/yarn
+        key: eslint-plugin-ember-${{ hashFiles('yarn.lock') }}
+        restore-keys: eslint-plugin-ember
```

Comment:
FYI `setup-node` has built-in caching now: https://github.com/actions/setup-node#caching-packages-dependencies 😉

Reply:
Updated in #1312.